hkmc2/shared/src/test/mlscript-compile/apps/parsing-web-demo/index.html

```diff
@@ -0,0 +1,264 @@
+      border: 1px solid #000;
+      box-shadow: rgb(187, 187, 187) 2px 2px 0px 0px;
+    }
+    
+    .select:focus {
+      outline: none;
+      box-shadow: rgb(187, 187, 187) 2px 2px 0px 0px, 0 0 0 .1rem rgba(52, 130, 255, 0.84);
+    }
+
+    .button {
+      line-height: 1.5;
+      background: rgb(153, 197, 255);
+      border: 1px solid #000;
+      box-shadow: rgb(187, 187, 187) 2px 2px 0px 0px;
+    }
+
+    .button:focus {
+      box-shadow: rgb(187, 187, 187) 2px 2px 0px 0px, 0 0 0 .1rem rgba(0, 0, 0, .2);
+    }
+
+    collapsible-tree {
+      font-family: var(--monospace);
+    }
+
+    details {
+      margin-left: 1em;
+      cursor: pointer;
+    }
+
+    summary {
+      outline: none;
+      list-style: none;
+    }
+
+    details>summary::-webkit-details-marker {
+      display: none;
+    }
+
+    details:not([open]):not([leaf])>summary::after {
+      content: " ...";
+    }
+
+    collapsible-tree details {
+      position: relative;
+    }
+
+    collapsible-tree details>.rule {
+      position: absolute;
+      top: 1rem;
+      left: 1px;
+      bottom: 0;
+      border-left: 1px dashed rgba(0, 0, 0, .2);
+    }
+
+    collapsible-tree>details {
+      margin-left: 0;
+    }
+
+    figcaption {
+      font-style: italic;
+      font-weight: 600
+    }
+
+    figure:target figcaption {
+      color: #007bff
+    }
+
+    figure:target figcaption::before {
+      font-style: normal;
+      content: '⯈ '
+    }
+
+    figure:target figcaption::after {
+      font-style: normal;
+      content: ' ⯇'
+    }
+
+    svg.railroad-diagram {
+      /* background-color: hsl(30,20%,95%); */
+      background: transparent;
+    }
+
+    svg.railroad-diagram path {
+      stroke-width: 2;
+      stroke: rgba(1, 1, 1, 0.80);
+      fill: transparent;
+    }
+
+    svg.railroad-diagram text {
+      text-anchor: middle;
+      white-space: pre;
+    }
+
+    svg.railroad-diagram text.diagram-text {
+      font-size: 12px;
+    }
+
+    svg.railroad-diagram text.diagram-arrow {
+      font-size: 16px;
+    }
+
+    svg.railroad-diagram text.label {
+      text-anchor: start;
+    }
+
+    svg.railroad-diagram text.comment {
+      font: italic 12px monospace;
+    }
+
+    svg.railroad-diagram g.non-terminal text {
+      font-style: -apple-system, system-ui, sans-serif;
+      font-weight: 600;
+      font-style: italic;
+    }
+
+    svg.railroad-diagram g.terminal text {
+      font-family: Inconsolata, monospace;
+    }
+
+    svg.railroad-diagram rect {
+      stroke-width: 1.5;
+      stroke: black;
+      fill: #ffffff;
+    }
+
+    svg.railroad-diagram rect.group-box {
+      stroke: gray;
+      stroke-dasharray: 10 5;
+      fill: none;
+    }
+
+    svg.railroad-diagram path.diagram-text {
+      stroke-width: 1.5;
+      stroke: black;
+      fill: white;
+      cursor: help;
+    }
+
+    svg.railroad-diagram g.diagram-text:hover path.diagram-text {
+      fill: #eee;
+    }
+  </style>
+</body>
+
+</html>
```

Comment:
```suggestion
</html>

```

Reply:
Fixed in [1e7429a](https://github.com/hkust-taco/mlscript/pull/277/commits/1e7429adc0405bd6ade92c6ea4dcf612d70fc1a4)